cnnimg.py

```diff
@@ -31,10 +31,7 @@
 import Image as img
 import numpy as np
 import os.path
-import warnings
 
```

Comment:
In Python 2.x print is actually a special statement and not a function. This print behavior in Python 2 can be changed to that of Python 3 using:
```from __future__ import print_function```

PIL is no longer supported. It's popularly replaced by Pillow. I had been thinking to update the program to use Pillow instead. Would you be interested to do this?